src/java.desktop/share/classes/sun/print/PSPrinterJob.java

```diff
@@ -267,9 +267,6 @@ public class PSPrinterJob extends RasterPrinterJob {
    private double xres = PS_XRES;
    private double yres = PS_YRES;
 
-   /* non-null if printing EPS for Java Plugin */
-   private EPSPrinter epsPrinter = null;
-
```

Comment:
This looks fine, but what about all unused xx_MASK and xx_SHIFT and hexDigits at the start of this class? and unused escapeParens() below?

Reply:
I guess un-related to what I was deleting but I will add it in as these can't have been used in a long time.